test.js

```diff
@@ -34,4 +61,15 @@ function toArray (cb) {
       read(null, next) //loop again.
     })
   }
-}
\ No newline at end of file
+}
+
+function toArraySync(read){
```

Comment:
I'm not super comfortable with `toArraySync`, though I realise it cleans up the test suite a fair bit